query-tuning/query-optimization-examples.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Query Optimization Examples'
+backlink_href: /docs/query-tuning
+backlink_title: 'Query Tuning'
+---
+
+import DocsBetaFeature from '../components/DocsBetaFeature'
+
+<DocsBetaFeature feature="Query Tuning" />
+
+Query optimization often involves recognizing and addressing common patterns that lead
+to suboptimal performance. While seasoned database engineers may spot these patterns
+through experience, Query Tuning helps identify and solve them systematically. Each
+pattern has specific indicators that suggest its presence, methods to validate the
+issue, and proven approaches to resolve it.
+
+We'll explore three common patterns that often impact query performance.
+
+## Inefficient Nested Loops Pattern
```

Comment:
It would be nice to have a ToC (list) in the beginning of this page, so that people can jump to these 3 patterns